lib/meta-box-partial-page.php

```diff
@@ -402,3 +434,21 @@ function gutenberg_meta_box_post_form_hidden_fields( $post ) {
 	// Permalink title nonce.
 	wp_nonce_field( 'samplepermalink', 'samplepermalinknonce', false );
 }
+
+/**
+ * Admin action which toggles the 'enable_custom_fields' option, then redirects
+ * back to the editor. This allows Gutenberg to render a control that lets the
+ * user to completely enable or disable the 'postcustom' meta box.
+ *
+ * @since 5.2.0
+ */
+function gutenberg_toggle_custom_fields() {
+	check_admin_referer( 'toggle_custom_fields' );
+
+	$enable_custom_fields = get_option( 'enable_custom_fields', false );
+	update_option( 'enable_custom_fields', ! $enable_custom_fields );
+
+	wp_safe_redirect( wp_get_referer() );
```

Comment:
Needs an `exit` after `wp_safe_redirect()` to kill the process.

Reply:
👌fixed in fe4785d80.